src/lib/utilities/version-check.test.ts

```diff
@@ -21,6 +22,7 @@ describe('isRecommendedVersionNewer', () => {
     expect(isVersionNewer('xxx', '1.01.1')).toBe(false);
     expect(isVersionNewer('1.1', '1.01.1')).toBe(false);
     expect(isVersionNewer('1.01', '1.02')).toBe(false);
+    expect(isVersionNewer('1.19.3', '1.19')).toBe(false);
```

Comment:
Isn't `1.19.3` newer than `1.19` though?

Reply:
No not if you don't provide a patch. So if you want to check if it's => 1.20, you do isVersionNewer(version, '1.19').